pkg/upgrade/utils.go

```diff
@@ -0,0 +1,16 @@
+package upgrade
+
+import "strings"
+
+func flagBoolValue(v interface{}) bool {
+	strValue, isString := v.(string)
+	if isString {
+		if strings.EqualFold(strValue, "true") {
```

Comment:
You can just return the result of EqualFold:

```
if isString {
  return strings.EqualFold(strValue, "true")
}
```

But I wonder why you just don't use `strconv.ParseBool("true")`

Reply:
Done.